### PR TITLE
Touch-up matsci spacesuits sprites

### PR DESCRIPTION
**Pull request by blackrep**
[feature][cleanliness]

## About the PR 
This PR works the matsci spacesuits & associated code to do the following:
1. Make the end-result spacesuit worn and item sprites more inline with the material's true color
2. Make the helmet visor color visible on the worn sprite
3. Clean up the amount of item specific code in `Mat_FabRecipes.dm`

## Why's this needed? 
Messing around with darker materials such as: uqill, plasmasteel, dyneema, etc. always created a spacesuit that was just a black non-identifiable blobs when worn. These new lighter base sprites make it so that a lot of the matsci spacesuits are much more visible. Plus visor mats previously had no impact on the worn sprite, meaning your selected material had no impact on your fashion sense.

TLDR: I want fashionable matsci spacesuits.
#### Comparison
| Old | New |
| ---- | ----- |
| image | image |
#### Various Colors
image
#### 360 View
image
#### Various Colors
image

### CODE REVIEW

**Comment by ThatFiveGuys** [comment]
Finally, I can do proper amongus roleplay

File: code/obj/item/clothing/helmets.dm [hunk 1]
```diff
@@ -123,53 +123,55 @@
...
+		UpdateOverlays(visrItemImg, "item-visor")
+		UpdateOverlays(fabrItemImg, "item-helmet")
+		// Setup worn overlays
+		fabrWornImg.color = helmMat.color
+		visrWornImg.color = visrMat.color
+		src.wear_image.overlays += fabrWornImg
+		src.wear_image.overlays += visrWornImg
+		// Add back the helmet texture since we overide the material apparance
+		if (helmMat.texture)
+			src.setTexture(helmMat.texture, helmMat.texture_blend, "material")
```
Comment (blackrep) [inline code/obj/item/clothing/helmets.dm:174]: This recolors the base helmet. However I wasn't able to figure out how to set the texture on the visor item / worn sprite overlays. If anyone has any pointers on how to add this, I'll gladly add it to this PR.

**Review by ZeWaka — APPROVED** [review]
as much as i want actual 3-4 color mapping instead of raw simple .color this is fine for now
